File: C#/Command/AddCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Command
{
    class AddCommand : ICommand
    {
        private SimpleCalc calculator;
        private List<double> digits;

        public AddCommand(SimpleCalc calculator, List<double> digits)
        {
            Console.WriteLine("COMMAND: Creating command");
            this.calculator = calculator;
            this.digits = digits;            
        }

        public void Execute()
        {
            Console.WriteLine("COMMAND: Executing command from invoker, delgating to logic");
            calculator.Sum(digits);
        }
    }
}

File: python/Command/AddCommand.py
from ICommand import ICommand
from SimpleCalc import SimpleCalc

class AddCommand:    
    def __init__(self, calculator: SimpleCalc, digits: list[float]) -> float:
        print("COMMAND: Creating command\n")
        self.calculator = calculator
        self.digits = digits

    def Execute(self) -> float:
        print("COMMAND: Executing command from invoker, delegating to logic\n")
        self.calculator.Sum(self.digits)